grails-app/services/org/broadinstitute/orsp/IssueService.groovy

```diff
@@ -489,4 +489,15 @@ class IssueService implements UserInfo {
         IssueType.valueOfName(issue?.getType())?.prefix?.toLowerCase()
     }
 
+    /**
+     * Project can have PM ACTOR CREATOR.
+     * CG only have CREATOR.
+     * @param extraProperties
+     * @return
+     */
+    static Collection<String> getPmOrActor(Map<String, List<String>> extraProperties) {
+        Collection<String> userPm = extraProperties.findAll ({ it.key == IssueExtraProperty.PM }).values().flatten()
```

Comment:
userPm name seems to indicate a single item but this is a Collection. Probably should be named differently.